.changeset/rotten-hairs-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+aria status description is now accurate
```

Comment:
It's usually helpful to mention which component you're changing in the changeset
```suggestion
TreeView: aria status description is now accurate
```